Gems/PhysX/Code/Editor/Source/MaterialEditor/Window/MaterialEditorWindow.cpp

```diff
@@ -0,0 +1,33 @@
+
+#include <AzCore/Utils/Utils.h>
+
+#include <QLabel>
+#include <QVBoxLayout>
+
+#include <Editor/Source/MaterialEditor/Window/MaterialEditorWindow.h>
+
+namespace PhysX
+{
+    MaterialEditorWindow::MaterialEditorWindow(QWidget* parent)
+        : QWidget(parent)
+    {
+        QVBoxLayout* mainLayout = new QVBoxLayout(this);
+
+        QLabel* introLabel = new QLabel("Put your cool stuff here PHYSX!", this);
+        mainLayout->addWidget(introLabel, 0, Qt::AlignCenter);
```

Comment:
This is the physx window, very simple label too for now. But in the future it'll grow to look like the render material editor. Since each gem is independent, we can decide how to add functionality here, it can be totally independent classes or it can reuse as much as possible from render material editor.